ext/zip/php_zip.c

```diff
@@ -30,18 +30,6 @@
 #include "php_zip.h"
 #include "php_zip_arginfo.h"
 
-/* zip_open is a macro for renaming libzip zipopen, so we need to use PHP_NAMED_FUNCTION */
```

Comment:
I don't understand this comment. Is this still valid?

Reply:
cc @remicollet

Reply:
I've requested a review from him ^^

Reply:
No, it's no longer relevant.

Reply:
For reference, this has been fixed in #5194.

Reply:
Not this comment didn't apply anymore (and for years)